### PR TITLE
math32: Add basic support for ARM64

### PR DESCRIPTION
**Pull request by blackrez**
I add a basic instructions for ARM64, a little step for the big issue #1200.
I also remove TODO in AMD64.

### CODE REVIEW

File: internal/math32/sqrt.go [hunk 1]
```diff
@@ -17,7 +17,9 @@ import (
 //	Sqrt(±0) = ±0
 //	Sqrt(x < 0) = NaN
 //	Sqrt(NaN) = NaN
-func Sqrt(x float32) float32 {
```
Comment (Kunde21) [inline internal/math32/sqrt.go]: This is the version that is used when asm isn't available.  
Use build tags to avoid duplicate definitions.  `!amd64 !arm64 noasm appengine safe`

**Comment by kortschak** [comment]
While I think that having `GOARCH=arm64` asm is valuable, I am concerned about the lack of our ability to integrate this work with continuous integration. Are there services that provide ARM64 architecture testing?

File: internal/math32/sqrt_amd64.s [hunk 2]
```diff
@@ -8,8 +8,7 @@
 
 //+build !noasm,!appengine,!safe
 
-// TODO(kortschak): use textflag.h after we drop Go 1.3 support
```
Comment (kortschak) [inline internal/math32/sqrt_amd64.s]: Please revert this. It needs to be done, but it should be in a separate PR.

File: internal/math32/sqrt_arm64.s [hunk 3]
```diff
@@ -6,15 +6,14 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build !noasm,!appengine,!safe
+//+build !noasm,!appengine,!safe
 
-package math32
+#include "textflag.h"
+// Don't insert stack check preamble.
```
Comment (kortschak) [inline internal/math32/sqrt_arm64.s]: Please delete this comment.

File: internal/math32/sqrt_arm64.s [hunk 4]
```diff
@@ -6,15 +6,14 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build !noasm,!appengine,!safe
+//+build !noasm,!appengine,!safe
```
Comment (kortschak) [inline internal/math32/sqrt_arm64.s]: Please revert this, the conventional syntax is `// +build <tags>`.

**Comment by kortschak** [comment]
There are: https://blog.travis-ci.com/2019-10-07-multi-cpu-architecture-support.

See #1217.

There is a non-trivial number of failures there, all to do with float precision differences between amd64 and arm64.

**Comment by kortschak** [comment]
When you resolve the conflicts here, please rebase onto master where we are now testing on arm64.
